examples/gallery/plot/double-y-axes.py

```diff
@@ -0,0 +1,58 @@
+"""
+Double Y-axes
+-------------
+
+The ``frame`` parameter of the plotting methods of the :class:`pygmt.Figure`
+class can control which axes should be plotted and possibly show annotations
+and tick marks. By default, all 4 axes are plotted, along with annotations
```

Comment:
Just a note that this is planned to change in 6.2.

Reply:
So we have to explicitly indicate all 4 axes, otherwise, they won't be plotted?

Reply:
The (planned) new modern default is to draw all four axes and annotate only the left and bottom (WrStZ). See here:
https://gmt-f853flcr1-gmt.vercel.app/theme-settings.html